source/write/insert.txt

```diff
@@ -0,0 +1,176 @@
+.. _c-write-insert:
+
+================
+Insert Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, write, save, create
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to add
+documents to a MongoDB collection by performing **insert operations**.
+
+An insert operation inserts one or more documents into a MongoDB collection.
+You can perform an insert operation by using the following methods:
```

Comment:
C doesn't have methods, only functions. Applies to all occurrences on this page.

```suggestion
You can perform an insert operation by using the following methods:
```